.github/workflows/continous-integration-docker.yml

```diff
@@ -560,6 +560,7 @@ jobs:
       test_id: generate-checkpoints-testnet
       test_description: Generate Zebra checkpoints on testnet
       test_variables: '-e NETWORK=Testnet -e GENERATE_CHECKPOINTS_TESTNET=1 -e ZEBRA_FORCE_USE_COLOR=1 -e ZEBRA_CACHED_STATE_DIR=/var/cache/zebrad-cache'
+      network: "Testnet"
```

Comment:
I wonder why this is not an issue with `full-sync-to-tip-testnet` or should the fix be applied there too?

Reply:
Is there a way to find any recent runs of that job? I would check the logs for it.

Reply:
It looks like there are no recent runs of that job. I think it is likely it will have the same issue.

Reply:
Not an easy way, sadly (I'll be refactoring this to be easier in the following sprints).

But I've just realized we must apply this fix on `full-sync-to-tip-testnet` as we're saving to the disk there, based on this parameter `saves_to_disk: true`.

Which then would corrupt the `Mainnet` disk if this parameter is not specified with the right correct Network:
https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/deploy-gcp-tests.yml#L1734-L1739

Reply:
Done in [16899b9](https://github.com/ZcashFoundation/zebra/pull/7309/commits/16899b97a7e49a2d594e41f22a62b9218bc42497).